src/Physics/Evaluate_friction_law.f90

```diff
@@ -1249,13 +1254,11 @@ SUBROUTINE rate_and_state_nuc103(TractionGP_XY,TractionGP_XZ,        & ! OUT: tr
 
     !TU 7.07.16: if the SR is too close to zero, we will have problems (NaN)
     !as a consequence, the SR is affected the AlmostZero value when too small
-    AlmostZero = 1d-45
+    AlmostZero = 1d-8
```

Comment:
this should remain 1d-45, right?

Reply:
those are the values that worked for TP. Alice also doubted why we need such a low value.

Reply:
The question is rather why one needs an arbitrary value here which is not constrained by anything....